cmd/kube-apiserver/app/options/options.go

```diff
@@ -235,5 +236,7 @@ func (s *ServerRunOptions) Flags() (fss apiserverflag.NamedFlagSets) {
 	fs.StringVar(&s.ServiceAccountSigningKeyFile, "service-account-signing-key-file", s.ServiceAccountSigningKeyFile, ""+
 		"Path to the file that contains the current private key of the service account token issuer. The issuer will sign issued ID tokens with this private key. (Requires the 'TokenRequest' feature gate.)")
 
+	fs.AddGoFlagSet(goflag.CommandLine)
```

Comment:
This seems a bit redundant as we are already doing it https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-apiserver/apiserver.go#L47 for the stand-alone KAS binary.

Reply:
In #64517, we introduced logical sections, and with that, we no longer read from `pflag.CommandLine` for the list of flags. #68054 introduced some changes in `apiserver/pkg/util/flag/`.  I'm waiting for that to be merged. See #70204. Sorry! I should probably close this PR since I have broken it down into three individual PRs.